tests/console/systemd_resolved.pm

```diff
@@ -0,0 +1,79 @@
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: systemd_resolved
+# Summary: Check that the network daemon in use is the expected one
+# - setup the nss resolved in /etc/nsswitch.conf
+# - test some DNS queries
+# - test systemd-resolved with DNSOverTLS and DNSSEC
+# - setting up systemd resolved locally, switch /etc/resolv.conf to it
+# Maintainer: qe-core <qe-core@suse.com>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use utils;
+use version_utils;
+
+sub enable_nm {
```

Comment:
I guess enable_nm would make more sense if it would be called cleanup or teardown